### PR TITLE
PD-852: Fix glyph flexbox issue

### PR DESCRIPTION
**Pull request by DesignerDave**
## ✍️ Proposed changes

Fixes an issue where glyphs would shrink in flex containers when consumed individually.

🎟 _Jira ticket:_ [PD-852](https://jira.mongodb.org/browse/PD-852)

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] ~I have added tests that prove my fix is effective or that my feature works~
- [ ] ~I have added necessary documentation (if appropriate)~
- [x] I have run `yarn changeset` and documented my changes

### CODE REVIEW

File: packages/icon/src/template.ts [hunk 1]
```diff
@@ -73,6 +74,11 @@ module.exports = function template(
         color: \${fill};
       \`;
 
+      const noFlexSizing = css\`
```
Comment (brookescarlett) [inline packages/icon/src/template.ts:77]: Why are we setting flex-grow to 0? IIRC, don't we find ourselves adding that property to maintain the icon's size

Comment (DesignerDave) [reply packages/icon/src/template.ts:77]: You're right, this should be the implicit value already. By being explicit here, I hoped to make it easier to debug issues around this related to setting that value to something other than `0`.

If you feel strongly, I can remove this.

Comment (brookescarlett) [reply packages/icon/src/template.ts:77]: I thought we had discussions around setting this to 1 explicitly rather than setting to 0 explicitly.

Comment (DesignerDave) [reply packages/icon/src/template.ts:77]: Fair point, I'll remove.

**Comment by lazytype** [comment]
I think you need to run `yarn pre-build` again

**Comment by DesignerDave** [comment]
> I think you need to run `yarn pre-build` again

Will do. In general, how can we adjust our actions to fix this in the future?

**Comment by lazytype** [comment]
> > I think you need to run `yarn pre-build` again
> 
> Will do. In general, how can we adjust our actions to fix this in the future?

I think the simplest approach is to have the action run the pre-build command and use git to check no files have changed

**Comment by DesignerDave** [comment]
> I think the simplest approach is to have the action run the pre-build command and use git to check no files have changed. This can also be used to check when `yarn.lock` needs to be updated too.

Sounds good! Would you mind making a ticket for that with the details you have? I don't think it should be a priority now, but we should consider doing that for our quality of life 😅

**Review by brookescarlett — APPROVED** [review]
lgtm

**Pull request merged**